include/aws/common/config.h.in

```diff
@@ -25,5 +25,6 @@
 #cmakedefine AWS_HAVE_GCC_OVERFLOW_MATH_EXTENSIONS
 #cmakedefine AWS_HAVE_GCC_INLINE_ASM
 #cmakedefine AWS_HAVE_MSVC_MULX
+#cmakedefine AWS_HAVE_EXECINFO
 
 #endif
```

Comment:
do we need a newline here?

Reply:
It was there before, so I left it. I don't think so, looking at the output. Does it offend you so? :)

Reply:
Oh I see, you meant at the end. I've added one.